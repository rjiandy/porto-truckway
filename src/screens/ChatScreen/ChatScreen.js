import React from 'react';

import {
  Body,
  Page,
  Section,
  StyledText
} from '../../components';

function ChatScreen(props) {
  return (
    <Page>
      <Body>
        <Section>
          <StyledText>ChatScreen</StyledText>
        </Section>
      </Body>
    </Page>
  );
}

export default ChatScreen;
