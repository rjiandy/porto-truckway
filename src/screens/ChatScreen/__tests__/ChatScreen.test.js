/**
 * @jest-environment jsdom
 */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import ChatScreen from '../ChatScreen';

test('renders correctly', () => {
  const chat = renderer.create(<ChatScreen />).toJSON();
  expect(chat).toMatchSnapshot();
});
