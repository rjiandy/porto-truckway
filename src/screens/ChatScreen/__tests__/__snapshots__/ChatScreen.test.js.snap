// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "height": 30,
      }
    }
  />
  <RCTScrollView>
    <View>
      <View>
        <Text
          style={
            Array [
              undefined,
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ]
          }
        >
          ChatScreen
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
